src/main/java/org/reactome/orthoinference/SkipInstanceChecker.java

```diff
@@ -25,7 +25,7 @@
 	private static Set<String> skipList = new HashSet<>();
 
 	// Skiplist was traditionally provided in a file, but since it's currently just 3 instances, I've just hard-coded them here.
-	public static void getSkipList(String pathToSkipList) throws NumberFormatException, Exception
+	public static void getSkipList(String pathToSkipList) throws Exception
 	{
 		String[] pathwayIdsToSkip = {"162906","168254","977225"};
```

Comment:
I noticed for these instance ids, could they have named constants so the pathways are explicit?

Reply:
The middle ID, `168254`, isn't valid anymore in our current DB. It was for Influenza Infection when I checked in v65. That matches `168255` in the current database, so I updated that DBID.